controllers/operator.go

```diff
@@ -395,6 +400,22 @@ func (o *operator) updateDeploy(deploy *appsv1.Deployment) (*appsv1.Deployment,
 	return deploy, nil
 }
 
+func (o *operator) managePodHealth(cl client.Client, conditions ...string) error {
```

Comment:
Function name is a bit misleading. It does the only thing - deletes pods in unknown state. It has nothing to do with health checks or other possible statuses. So I'd prefer to have something `DeleteStuckedPods`, `DeleteUnknownStatePods` or smthng.

Reply:
Changed to `deletePodsWithStatusUnknown`.